lib/sonos-http-api.js

```diff
@@ -67,7 +67,27 @@ function HttpAPI(discovery, settings) {
     }
 
     const params = req.url.substring(1).split('/');
-    
+
+    // Change node environment log level on request. 
+    if (/^\/loglevel\/.+$/.test(req.url)) {
+        let param = params[1];
+        switch (param) {
+        case 'off':
+            param = 'error';
```

Comment:
Hm, I don't follow what `off` would actually set. I would also like that just getting `/loglevel` would actually return the current loglevel.